package.json

```diff
@@ -60,28 +59,24 @@
         "babel-loader": "~8.2.2",
         "babel-plugin-const-enum": "^1.1.0",
         "blob-polyfill": "^6.0.20211015",
-        "copy-webpack-plugin": "~9.0.1",
         "copyfiles": "^2.4.1",
         "cross-env": "7.0.3",
-        "css-loader": "^6.7.3",
         "esbuild": "^0.12.25",
         "esbuild-jest": "^0.5.0",
         "eslint": "^7.19.0",
         "eslint-config-prettier": "^6.11.0",
         "eslint-plugin-react": "^7.22.0",
-        "html-webpack-plugin": "^5.3.2",
         "husky": "^1.3.1",
         "jest": "^27.1.0",
         "lint-staged": "^10.5.4",
-        "mini-css-extract-plugin": "^2.1.0",
         "npm-run-all": "^4.1.5",
         "nyc": "~15.0",
         "postcss": "^8.2.10",
         "postcss-cli": "^8.3.1",
         "postcss-preset-env": "^6.7.0",
         "prettier": "^2.2.1",
-        "react": "^16.9.0",
-        "react-dom": "^16.9.0",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
```

Comment:
So theoretically you can remove all traces of webpack from this package.json now that only the example app needs it

Reply:
oh you're right, I thought we were using that `webpack.config.js` for something, so I didn't remove it but I guess not, I'll make another ticket to do that.